File: Programming_Fundamentals/10.Arrays-Exercises/09.IndexOfLetters/IndexOfLetters.cs
using System;

namespace _09.IndexOfLetters
{
    class IndexOfLetters
    {
        static void Main(string[] args)
        {
            var input = Console.ReadLine().ToCharArray();
            foreach (var character in input)
            {
                Console.WriteLine($"{character} -> {((int)character) - 97}");
            }
        }
    }
}

File: Software Technologies/Java Basics - Exercises/src/IndexOfLetters.java
import java.util.Arrays;
import java.util.Scanner;

public class IndexOfLetters {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        char[] arrayOfLetters = scan.nextLine().toLowerCase().toCharArray();

        for (char letter: arrayOfLetters) {
            int numericValueOfLetter = Character.getNumericValue(letter) - 10;
            System.out.printf("%s -> %d%n", letter, numericValueOfLetter);
        }
    }
}
